src/helpers/apiToGameReleaseVersionMap.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Current [april 2024] is v81, which requires at least 0.4.7
```

Comment:
This wouldn't be bad as an inline comment above `if (apiVersion === 81)`, though the rest is self-explanatory in the code.

Reply:
moved inline and removed the rest.